client/signup/steps/site-topic/index.jsx

```diff
@@ -141,24 +128,24 @@ class SiteTopicStep extends Component {
 					subHeaderText={ commonSubHeaderText }
 					fallbackSubHeaderText={ commonSubHeaderText }
 					signupProgress={ this.props.signupProgress }
-					stepContent={ this.renderContent( topicLabel, commonPlaceholder ) }
+					stepContent={ this.renderContent( topicLabel ) }
 				/>
 			</div>
 		);
 	}
 }
 
 const mapDispatchToProps = ( dispatch, ownProps ) => ( {
-	submitSiteTopic: siteTopic => {
+	submitSiteTopic: ( { siteTopicSlug, siteTopicValue } ) => {
 		const { flowName, goToNextStep } = ownProps;
 
 		dispatch(
 			recordTracksEvent( 'calypso_signup_actions_submit_site_topic', {
-				value: siteTopic,
+				value: siteTopicSlug,
```

Comment:
`siteTopicSlug ` is not part of the state tree, so it will get an _undefined_ value here if you come to this step after hitting the Back button from later steps. Should we include `siteTopicSlug` too in the redux state tree?

Reply:
> so it will get an undefined value here if you come to this step after hitting the Back button from later steps

Nice catch. Thanks! I'll fix.